test/functionalTest/cases/1317_GET_v2_subscription/get_subscription.test

```diff
@@ -0,0 +1,249 @@
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create 2 subscriptions
+# 02. GET /v2/subscriptions/SUB_ID
+# 03. GET /v2/subscriptions/SUB_ID2
+#
+
+echo "01. Create 2 subscriptions"
+echo "=========================="
+payload='{
+  "entities": [
+    {
+      "type": "Room",
+      "isPattern": "false",
+      "id": "ConferenceRoom"
+    },
+    {
+      "type": "Room",
+      "isPattern": "false",
+      "id": "OfficeRoom"
+    }
+  ],
+  "attributes": [
+    "temperature",
+    "occupancy",
+    "lightstatus"
+  ],
+  "reference": "http://127.0.0.1:'${LISTENER_PORT}'/notify",
+  "duration": "P5Y",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+      "condValues": [
+        "temperature",
+        "timestamp"
+      ]
+    }
+  ],
+  "throttling": "PT5S"
+}'
+orionCurl --url /v1/subscribeContext --payload "${payload}" --json
+SUB_ID=$(echo "$_response" | grep subscriptionId |  awk -F ':' '{gsub(/[ \t",]+/, "", $2); print $2}' )
+echo
+echo
+
+payload='{
+  "entities": [
+    {
+      "type": "Car",
+      "isPattern": "false",
+      "id": "MyCar"
+    },
+    {
+      "type": "Room",
+      "isPattern": "false",
+      "id": "Garage"
+    }
+  ],
+  "attributes": [
+    "speed",
+    "location"
+  ],
+  "reference": "http://127.0.0.1:'${LISTENER_PORT}'/notify",
+  "duration": "P5Y",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+      "condValues": [
+        "speed"
+      ]
+    }
+  ]
+}'
+orionCurl --url /v1/subscribeContext --payload "${payload}" --json
+SUB_ID2=$(echo "$_response" | grep subscriptionId |  awk -F ':' '{gsub(/[ \t",]+/, "", $2);print $2}' )
+echo
+echo
+
+
+echo "02. GET /v2/subscriptions/$SUB_ID"
+echo "=================================================="
+orionCurl --url /v2/subscriptions/$SUB_ID --json
+echo
+echo
+
+
+echo "03. GET /v2/subscriptions/$SUB_ID2"
+echo "=================================================="
+orionCurl --url /v2/subscriptions/$SUB_ID2 --json
+echo
+echo
+
+
+--REGEXPECT--
+01. Create 2 subscriptions
+==========================
+HTTP/1.1 200 OK
+Content-Length: 135
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "subscribeResponse": {
+        "duration": "P5Y",
+        "subscriptionId": "REGEX([0-9a-f]{24})",
+        "throttling": "PT5S"
+    }
+}
+
+
+HTTP/1.1 200 OK
+Content-Length: 108
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "subscribeResponse": {
+        "duration": "P5Y",
+        "subscriptionId": "REGEX([0-9a-f]{24})"
+    }
+}
+
+
+02. GET /v2/subscriptions/REGEX([0-9a-f]{24})
+==================================================
+HTTP/1.1 200 OK
+Content-Length: 449
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "expires": "REGEX(\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d.00Z)",
+    "id": "REGEX([0-9a-f]{24})",
+    "notification": {
+        "attributes": [
+            "temperature",
+            "occupancy",
+            "lightstatus"
+        ],
+        "callback": "http://127.0.0.1:9997/notify",
```

Comment:
I have realized that we have a bunch of them:

```
#grep -R notify test/functionalTest/cases/* | grep 999 | wc -l
20
```
So, this is not easly fixeable in this PR. I'll open an issue about it.